app/src/main/res/drawable/ic_delete.xml

```diff
@@ -5,5 +5,5 @@
         android:viewportHeight="24.0">
     <path
         android:pathData="M6,19c0,1.1 0.9,2 2,2h8c1.1,0 2,-0.9 2,-2V7H6v12zM19,4h-3.5l-1,-1h-5l-1,1H5v2h14V4z"
-        android:fillColor="#ffffff"/>
+        android:fillColor="@drawable/state_colors"/>
```

Comment:
Are color state lists supported in vector drawables? On a Nexus 5 / 6.0 the trashcan icon isn't getting set to the !isEnabled drawable listed in the state color list, even though the states are being updated correctly. I wonder if state color lists need to wrap individual drawables/svgs, rather than the other way around.